### ISSUE TITLE
some buildings result in console error and do not display

### ISSUE THREAD

**Issue by kfarr** [3DStreet/3dstreet#284]
Open this URL:
https://github.3dstreet.org/#https://streetmix.net/kfarr/100/university-blvd

Note the buildings do not load and console error:
```
caught TypeError: Cannot read properties of undefined (reading '0')
    at t.exports.createBuildingsArray (aframe-street-component.js:1:67011)
    at aframe-street-component.js:1:22698
    at aframe-street-component.js:1:22945
    at Array.forEach (<anonymous>)
    at t.exports.processBuildings (aframe-street-component.js:1:22056)
    at n.update (aframe-street-component.js:1:72294)
    at n.callUpdateHandler (component.js:417:10)
    at n.updateProperties (component.js:298:12)
    at d.updateComponent (a-entity.js:466:17)
    at d.setAttribute (a-entity.js:686:10)
```

Other scenes seem to work correctly so I guess it is one of these specific buildings that is causing the error?

**Comment by Algorush** [comment]
Yes, i had noticed about this issue earlier in one of issue thread. This happens because there is no support for 'parking-lot' in the building generation code, or rather not completely. I will make it

**Comment by kfarr** [comment]
closed with #285
